src/core_functions/scalar/enum/enum_functions.cpp

```diff
@@ -28,7 +28,12 @@ static void EnumRangeFunction(DataChunk &input, ExpressionState &state, Vector &
 	for (idx_t i = 0; i < enum_size; i++) {
 		enum_values.emplace_back(enum_vector.GetValue(i));
 	}
-	auto val = Value::LIST(enum_values);
+	Value val;
+	if (!enum_values.empty()) {
+		val = Value::LIST(enum_values);
```

Comment:
Perhaps we can use `Value::LIST(LogicalType::VARCHAR, enum_values)` instead of using `Value::EMPTYLIST?